docker/docker-compose.yml

```diff
@@ -576,9 +612,9 @@ services:
   #    - ns datawaveUsers
   #    - m.entries
   management-center:
-    profiles:
-      - management
-      - full
+#    profiles:
+#      - management
+#      - full
```

Comment:
You probably commented these while testing, but they should stay uncommented for your PR.